### PR TITLE
docker: update to use Node.js 16

### PR DESCRIPTION
**Pull request by BethGriggs**

### CODE REVIEW

**Comment by BethGriggs** [comment]
@yangcao77, we'd like to update the underlying Node.js version to Node.js 16. I've bumped the image versions, and done a semver major bump of the DevFile version. Could you review/let me know if there's anything else I need to update to make this change?

File: devfile.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 schemaVersion: 2.2.0
 metadata:
   name: nodejs
-  version: 1.0.1
+  version: 2.0.0
   displayName: Node.js Runtime
   description: Stack with Node.js 14
```
Comment (yangcao77) [inline devfile.yaml]: also update the description here
```suggestion
  description: Stack with Node.js 16
```

**Review by yangcao77 — COMMENTED** [review]
Everything look good, just the description need to be updated.

**Review by yangcao77 — APPROVED** [review]
lgtm

**Pull request merged**